### ISSUE TITLE
Font style "Underline"

### ISSUE THREAD

**Issue by SkyZero1228** [AvaloniaUI/Avalonia#2548]
How can I set the font style "Underline"?

**Comment by Gillibald** [comment]
That isn't supported yet but will be soon. Probably in the next release.

**Comment by derekantrican** [comment]
I see a PR was merged - is this possible now or are we still waiting on the next release?

**Comment by maxkatz6** [comment]
@derekantrican thanks.
Yes, it is possible for TextBlock in 0.10-preview with "<TextBlock Text="Email" TextDecorations="Underline" />".

**Comment by Dorin-Foaltin** [comment]
Is there a way to set the text to **underline** for a **TextBox** or **TextPresenter** as well?

I need the text from a TextBox to be underline as well, and the TextBox currently doesn't have the TextDecorations property. 

I also tried to change the style of the TextBox to underline the TextBlock that is rendering the text, but I found out that the text is rendered via a TextPresenter, which also doesn't have a TextDecorations property.
